tools/health/health_checker_pool.go

```diff
@@ -158,7 +161,7 @@ func (ncp *NodeHealthCheckerPool) CheckHeight(expectedHeight uint64, nodeHealthC
 	}
 }
 
-func (ncp *NodeHealthCheckerPool) WaitHeightAll(expectedHeight uint64) error {
+func (ncp *NodeHealthCheckerPool) WaitHeightAll(expectedHeight uint64, timeout time.Duration) error {
```

Comment:
why do we need one more timeout? There is already exist one ticker and the fun calls the ncp.CheckHeight method.